Switch away from the more limited distutils package to setuptools.
This patch has not yet been submitted upstream.

--- python-mimeparse-0.1.4/setup.py.orig   2014-03-25 14:33:23.587458089 -0600
+++ python-mimeparse-0.1.4/setup.py    2014-03-25 14:33:28.216518310 -0600
@@ -1,6 +1,6 @@
 #!/usr/bin/env python

-from distutils.core import setup
+from setuptools import setup
 import mimeparse

 setup(
